src/nvim/api/vim.c

```diff
@@ -2767,3 +2768,29 @@ void nvim__set_luahl(DictionaryOf(LuaRef) opts, Error *err)
 error:
   clear_luahl(true);
 }
+
+/// Search the helptags for a pattern
+///
+/// @param pattern the pattern to search
+/// @param[out] err error if any
+Array nvim_get_helptags(String pattern, Error *err)
```

Comment:
Do we want this to be `nvim_get_tags` already? Supporting only helptags is OK for now, but should it be spelled as `nvim_get_tags("pattern", {source="help"})` or something like that, so it is easier to extend later?

Reply:
That sounds like a good idea.
The thing that needs to be adressed too is the return value of the function.
After some discussion with @tjdevries we thought that a `taglist()`-like return value can be great, looking like this :
```
[
  {
    tag : String,
    fname : String,
    lnum : Integer
  }
]
```